docs/semgrep-ci/network-broker.md

```diff
@@ -90,10 +90,10 @@ Ensure that you are logged in to the server where you want to run Semgrep Networ
 2. Update the `config.yaml` by replacing `YOUR_BASE_URL` with your Bitbucket Data Center, GitLab, or GitHub URL:
 
   ```yaml
-  # for Bitbucket
+  # for Bitbucket - compatible with Network Broker versions 0.20.0 and later
```

Comment:
suggestion: do we want to link to the semgrep network broker docs here?

Reply:
this is on the network broker doc, unless you meant the network broker repo readme?

Reply:
nvm, missed that it was on the network broker file! LGTM